### DISCUSSION TITLE
Error R10 (Boot timeout)

### DISCUSSION THREAD

**Discussion by karimawi** [eritislami/evobot#567] [Q&A] [answered]
Every time I start the bot it crashes giving me this error:
![16110101223829154346595887197367](https://user-images.githubusercontent.com/69881381/104969064-3324b980-59f0-11eb-8831-70bf7453f988.jpg)

**Comment by Takiyo0** [comment, marked as answer]
Use worker instead of web. It only active for some minutes just like glitch.

**Reply by karimawi** [reply]
I tried using the console "npm start" and it crashes actually faster 🙂, I don't know what's a worker I'm dumb 😅

**Reply by karimawi** [reply]
I tried "node index.js" and it also crashed

**Reply by karimawi** [reply]
I also turned of the web and turned the work on... also crashes

**Reply by Takiyo0** [reply]
oh, you have to disable the web and enable worker only. It must disable the web because in your error, there is `Web Process Failed`

**Reply by karimawi** [reply]
I did that, it also crashes

**Reply by karimawi** [reply]
![Screenshot_20210119-141046_Chrome](https://user-images.githubusercontent.com/69881381/105032989-32c40700-5a60-11eb-97c5-2b879d8034da.jpg)

**Reply by karimawi** [reply]
Ok, I just removed the config vars and made my fork private so I can add my vars to it directly and it worked
